storage/backend/null/null.go

```diff
@@ -2,10 +2,20 @@
 
```

Comment:
Do we need these changes on this branch? The previous commit was disabling these.